standard/bridge-v1/emulator/main.go

```diff
@@ -181,8 +206,5 @@ func postMetricToDatadog(ctx context.Context, client *datadog.APIClient, metricN
 		Series: []datadog.MetricSeries{series},
 	}
 	_, _, err := client.MetricsApi.SubmitMetrics(ctx, payload)
-	if err != nil {
-		log.Fatal().Err(err).Msg("Failed to post metric to Datadog")
-	}
-	log.Debug().Msgf("Posted metric %s with value %f and tags %v", metricName, value, tags)
+	return err
```

Comment:
Nice change 👍 seems better the emulator service doesn't exit upon failed dd call